cms/static/js/programs/routers/core_app_program_router.js

```diff
@@ -0,0 +1,67 @@
+define([
```

Comment:
I don't think this needs to be in its own directory anymore. Since the core/dev distinction doesn't exist anymore, you should be able to move it to `cms/static/js/programs/router.js`.